docs/appendices/release-notes/5.5.1.rst

```diff
@@ -49,4 +49,5 @@ See the :ref:`version_5.5.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Added restriction to not allow to drop a column which participates in a
+  :ref:`table level constraint with other columns <check_constraint_multiple_cols>`.
```

Comment:
What happen currently, is the column dropped and the table is then in an unusable state? Or will this end in an non-clear other error?

Reply:
You get a weird error `Column b uknown` for example, because the constraints remain in metadata and it is not resolved to the internal oid based name, but nothing breaks, there is no cluster state update happening, the command just fails, so in the `DropColumnTask` we need to remove those constraints as well.

Reply:
Real fix: https://github.com/crate/crate/pull/14991